packages/gatsby-cli/src/reporter/patch-console.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * This module is used to patch console through our reporter so we can track
+ * these logs
+ */
+import util from "util"
+import { reporter as gatsbyReporter } from "./reporter"
+
+export const patchConsole = (reporter: typeof gatsbyReporter): void => {
+  console.log = (format: any, ...args: any[]): void => {
```

Comment:
According to https://nodejs.org/docs/latest-v8.x/api/util.html#util_util_format_format_args

Should we change that to `format: string, ...args: unknown[]` ?

Reply:
It's not exactly unknown though, it really is any. you can log anything to console. It is a weird situation though where we are highjacking console.* - which already have type signatures. This file is relatively small that i don't think we need to stress over, nor will we really get it entirely accurate. It was sort of a poor api design from the beginning.

Reply:
Yeah, makes sense :)